### ISSUE TITLE
GDevelop update changes AppImage name and breaks desktop shortcut

### ISSUE THREAD

**Issue by sebastienaubry** [4ian/GDevelop#1784]
Hi,
Under Linux Mint Cinnamon 19.3, I have downloaded the GDevelop AppImage and made a shortcut to it on the desktop.
When GDevelop updates itself, the AppImage filename is changed, e.g. from `GDevelop-5-5.0.0-beta93.AppImage` to `GDevelop-5-5.0.0-beta95.AppImage` (93 to 95).
This breaks the shortcut. Is there any way to prevent this behaviour and make it still work?
Thanks

**Comment by 4ian** [comment]
That would be a good questions for folks working on AppImage and/or electron-builder (used for packaging GDevelop on desktop) - I'm not super familiar with how it works to be honest. The shortcut is pointing to what exactly, the AppImage file?

**Comment by sebastienaubry** [comment]
Yes, the AppImage file is supposed to be made executable with `chmod +x`, so I made the shortcut point at it directly. 
I will look for an answer on the AppImage side... else I could write a minimal script to find the file and execute it (even \`ls ./*.AppImage\` works!). 
Another question would be: is it possible to add it automatically to the program menu.
Thanks

**Comment by Midhil457** [comment]
I don’t know if I am late. I used AppI ageLauncher to integrate it to the desktop. Shows up in the menu, desktop, etc.... Sorry, for replying to an old issue
